src/app/components/room/room.service.js

```diff
@@ -15,6 +15,11 @@
       'FeedConnection', 'DataChannelService', 'ActionService', 'jhConfig',
       'ScreenShareService'];
 
+  /**
+   * Service containing all rooms
```

Comment:
Well, that's not how I would describe this service. It has a function to get the list of rooms, but it is not a collection of items like, for example `FeedsService`.

I would rather say something like "Service to manage the communication with the Janus server".